entrypoint.d/999-odoo-wait

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [ "${WAIT_CONTAINER}" == false ]; then
+    exit 0
+fi
+
+log WARNING "Waiting for '${WAIT_CONTAINER}' to stop"
+
+while ping -c1 "${WAIT_CONTAINER}" &> /dev/null; do
+    log INFO "${WAIT_CONTAINER} responded to ping - waiting for exit."
```

Comment:
This should be `DEBUG`, or your logs can get full of these...
Also, I'd add a `sleep 1` so we wait 1s among pings.